### PR TITLE
UAR-1281 Add new field to capture the confirm removal statement acceptance

### PR DESCRIPTION
**Pull request by mwestacott**
### JIRA link

https://companieshouse.atlassian.net/browse/UAR-1281

### Change description

* New field added to DTO and DAO
* New container DTO and DAO classes added for the remove fields
* Test JSON updated to have the new container and field

### Work checklist

- [X] Tests added where applicable

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

**Comment by markpit** [comment]
does the /spec/schema.json need updating?

**Comment by mwestacott** [comment]
> does the /spec/schema.json need updating?

Yes, probably. Good point - I will look into this.

**Review by mattch1 — APPROVED** [review]
LGTM - now pending second approval.

File: spec/schema.json
Comment (mwestacott) [inline spec/schema.json:382]: `is_remove` field added in a previous PR but adding it to the schema was missed

**Review by markpit — APPROVED** [review]

**Pull request merged**
